.github/workflows/release-python-bindings.yml

```diff
@@ -59,6 +59,13 @@ jobs:
         with:
           args: --release --out dist -m client/bindings/python/Cargo.toml
```

Comment:
Only aarch64-apple-darwin is built for different versions ?

Reply:
It should detect the installed version, so probably can be also removed from aarch64-apple-darwin, do you want me to test?
You can see that it detects the correct version in the logs here https://github.com/Thoralf-M/iota.rs/actions/runs/3967181672/jobs/6798844991
> Found CPython 3.8 at /opt/hostedtoolcache/Python/3.8.16/x64/bin/python
> Found CPython 3.9 at /opt/hostedtoolcache/Python/3.9.16/x64/bin/python

Reply:
Better if it's consistent I think, yes